pnpm-lock.yaml

```diff
@@ -343,11 +342,10 @@ importers:
       '@octokit/plugin-retry': 3.0.9
       '@octokit/plugin-throttling': 3.5.2_@octokit+core@3.5.1
       '@popperjs/core': 2.11.4
-      '@senecacdot/satellite': 1.27.0
+      '@senecacdot/satellite': link:../../satellite
```

Comment:
cc @DukeManh, this shouldn't be happening, right?  Has that `.npmrc` change not landed yet?

Reply:
Looks like Kevan just picked up the change